app/services/gitlab_integration/project_creator.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module GitlabIntegration
+  class ProjectCreator
+    def initialize(controller_params, current_user)
+      @controller_params = controller_params
```

Comment:
I suggest to process controller_params right here and save them instead of controller_params